app/Entirety/entities/templates/entities/entity_list.html

```diff
@@ -12,23 +12,12 @@
                 <div class="p-3 border border-light rounded-pill bg-light">
                     <div class="input-group border-0 shadow-none">
                         <input name="search-id" type="text" class="form-control border-light rounded-pill"
-                               placeholder="Search entities by id"/>
-                        <span class="input-group-btn">
-                                        <button class="btn btn-info btn-lg bg-light border-light" type="submit"
-                                                value="search">
-                                            <i class="bi-search"></i>
-                                        </button>
-                                        </span>
-                    </div>
-                </div>
-            </form>
-        </div>
-        <div class="col-4">
-            <form action="">
-                <div class="p-3 border border-light rounded-pill bg-light">
-                    <div class="input-group border-0 shadow-none">
-                        <input name="search-type" type="text" class="form-control border-light rounded-pill"
-                               placeholder="Search entities by type"/>
+                               placeholder="Search entities by ......"/>
```

Comment:
Is it possible here to set a default selection, like "ID" for example?

Reply:
Otherwise, I think the user experience goes down than it used to, because the users could simply type in something and click the "Enter"